golang/frame_pool_b_test.go

```diff
@@ -0,0 +1,82 @@
+package tchannel_test
+
+import (
+	"math/rand"
+	"sync"
+	"sync/atomic"
+	"testing"
+
+	. "github.com/uber/tchannel/golang"
+)
+
+// Copyright (c) 2015 Uber Technologies, Inc.
```

Comment:
Good point, the license goes after package but before imports. Then I think goimports has been adding imports to above the license, and I didn't notice. Will fix in a separate PR, thanks.